neevaco/neeva-ios
# NYTimes TabGroup Case

Adds support for creating a tab group when a user navigates back to a tab that opened a child tab. (See test cases before or see the Tab Group paper doc for more info)

## Checklists

### How was this tested?

### Manual test cases
1:
- Create tab (nytimes.com)
- Navigate to link (nytimes.com/article)
- Create a new tab with original URL (nytimes.com)
- Check tab group created ✅

2: (tab already in tab group)
- Search new york times
- Open tab from SRP (nytimes.com) (creating tab group)
- Navigate to link (nytimes.com/article)
- Create a new tab with original URL (nytimes.com)
- Check nytimes.com inserted intro already existing tab group ✅

Everything else seems to perform normally (using `<a target=_blank ...>`) but LMK if there are other paths to try!

### Screenshots and screen recordings
1st case:

https://user-images.githubusercontent.com/37356984/157582039-9e8f8e45-1f5e-48c6-b5c4-cf4c58d4e6a8.mp4

2nd case:

https://user-images.githubusercontent.com/37356984/157582078-c61bdecf-d57f-4b46-94d0-597171f0edda.mp4
